### PR TITLE
Update Proving Control and Binding section.

### PR DESCRIPTION
**Pull request by msporny**
Partial editorial cleanup to appendices tracked as issue #728. This also addresses issue #583.

This section was a bit of a mess and required a fairly significant rewrite... I tried to save what I could, much of the language was quite old and represented thinking from 3+ years ago. I tried to update the language to be less prescriptive, point to sections that have been created to address each item, and remove the concept of "proving control over a key" -- 'cause that's not really a use case, it's just a smaller part of larger use cases such as authentication and authorization, which we already have sections that go to lengths to talk about those mechanisms elsewhere in the spec (so I pointed there).

***
<a href="https://pr-preview.s3.amazonaws.com/w3c/did-core/pull/738.html" title="Last updated on May 22, 2021, 2:58 PM UTC (fe809af)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/did-core/738/ab18a18...fe809af.html" title="Last updated on May 22, 2021, 2:58 PM UTC (fe809af)">Diff</a>

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -4391,127 +4391,61 @@ <h2>Choosing DID Resolvers</h2>
...
-It can be useful to express a binding of <a>DID</a> to a person's or company's
-real world identity, in a way that is provably asserted by a trusted authority
-such as a government. This can enable interactions that can be considered
-legally valid under one or more jurisdictions; establishing such bindings has to
-be carefully balanced against privacy considerations (see <a
-href="#privacy-considerations"></a>).
+It can be useful to express a binding of a <a>DID</a> to a person's or
+organization's physical identity in a way that is provably asserted by a
+trusted authority, such as a government. This specification provides a
+<a>verification relationship</a> in <a href="#assertion"></a> for these
```
Comment (rhiaro) [inline index.html:4436]:
```suggestion
the <a href="#assertion"></a> <a>verification relationship</a> for these
```

**Review by rhiaro — APPROVED** [review]

**Review by brentzundel — APPROVED** [review]

**Review by peacekeeper — APPROVED** [review]
I think this is good and makes it clearer what "control over a DID" means.

I do think it would still be valuable to keep some of the guidance about signed DID documents. We should say that some DID methods do that, and it can make sense depending on the DID method design, but signed DID documents alone are no guarantee that the DID document is the correct one for the DID.

We could merge this PR as-is, and then I could create another PR to add this back in (either as part of the same section, or a new section).

File: index.html [hunk 2]
```diff
@@ -4391,127 +4391,61 @@ <h2>Choosing DID Resolvers</h2>
...
 
         <p>
 The process of binding a <a>DID</a> to something in the real world, such as a
-person or a company, for example using <a>verifiable credentials</a> with the
-same subject as that <a>DID</a>, is out of scope for this specification. For
-more information, see the Verifiable Credentials Data Model [[VC-DATA-MODEL]].
+person or an organization, for example using <a>verifiable credentials</a> with
+the same subject as that <a>DID</a>, is contemplated by this specification
+and further defined in the Verifiable Credentials
+Data Model [[VC-DATA-MODEL]].
```
Comment (TallTed) [inline index.html:4448]:
```suggestion
The process of binding a <a>DID</a> to something in the physical world, such
as a person or an organization &mdash; for example, by using <a>verifiable
credentials</a> with the same subject as that <a>DID</a> &mdash; is
contemplated by this specification and further defined in the Verifiable
Credentials Data Model [[VC-DATA-MODEL]].
```

File: index.html (hunk 1 above)
Comment (TallTed) [inline index.html:4436]:
```suggestion
trusted authority, such as a government. This specification provides the
<a href="#assertion"></a> <a>verification relationship</a> for these
```

**Review by TallTed — CHANGES_REQUESTED** [review]
Focused more on the new text as itself, and less on whether the
new text carries everything it should from the old.

File: index.html (hunk 1 above)
Comment (msporny) [reply index.html:4436]: Applied in @rhiaro's suggestion.

File: index.html (hunk 2 above)
Comment (msporny) [reply index.html]: Applied in fe809af3db8f85296af1bcc49648efa470c572a4.

**Comment by msporny** [comment]
Editorial, multiple reviews, changes requested and made, no objections, merging.

**Comment by TallTed** [comment]
[Change applied](https://github.com/w3c/did-core/pull/738#discussion_r637413448) as `provides a the <a` still needs to lose the redundant and incorrect `a` article, which I could not suggest as a modification of the earlier suggestion from @rhiaro.

**Pull request merged**